app/controllers/interventions_controller.rb

```diff
@@ -2,7 +2,7 @@ class InterventionsController < ApplicationController
   def new
     @intervention_type = InterventionType.find(params[:intervention_type_id])
     @intervention_type_group = @intervention_type.intervention_type_group
-    @observation = current_user.school.observations.new(intervention_type_id: @intervention_type.id, at: Time.zone.now)
+    @observation = current_user.school.observations.new(intervention_type_id: @intervention_type.id)
```

Comment:
hi @ldodds  just to double check (as i may well be looking in the wrong place) but looks like there's a couple of validations for `:at` on `observation.rb`
```
  ...
  validates_presence_of :at, :school
  validate :at_date_cannot_be_in_the_future
  ...
```
will these need removing?

Reply:
No, they need to provide one. This just ensures they pick one rather than submitting the form with the default.

Reply:
👍